<?xml version="1.0"?>
<launch>

  <arg name="flight_with_gazebo" default="true"/>

  <!-- flight node  -->
  <group if="$(arg flight_with_gazebo)">
    <node name="ceo_mlcpp_flight_node" pkg="ceo_mlcpp" type="ceo_mlcpp_flight_node" output="screen">
      <param name="tsdf_voxels_per_side" value="8"/>
      <param name="voxel_resolution" value="0.4"/>
      <param name="max_ray_length_m" value="20.0"/>
      <param name="method" value="fast"/> <!-- simple, fast, merged -->
      <param name="color_mode" value="color"/>
    </node>
    <node name="visualizer2" pkg="rviz" type="rviz" output="screen" args="-d $(find ceo_mlcpp)/flight_rviz.rviz"/>
    <node name="visualizer3" pkg="ceo_mlcpp" type="pose_to_path.py"/>    
  </group>

</launch>